### PR TITLE
Add search past query to search.html

### PR DESCRIPTION
**Pull request by domenc**
issue #115 #86

### CODE REVIEW

File: piplmesh/frontend/static/piplmesh/css/style.css [hunk 1]
```diff
@@ -94,6 +94,10 @@ td.gsc-clear-button {
     display: none !important;
 }
 
+#google_query {
+    margin-bottom: -100px;
```
Comment (mitar) [inline piplmesh/frontend/static/piplmesh/css/style.css:98]: This looks really ugly? From where are those numbers?

File: piplmesh/frontend/templates/search.html [hunk 2]
```diff
@@ -5,6 +5,10 @@
 {% block title %}{% trans "Search" %}{% endblock %}
 
 {% block content %}
+    {% if "q" in request.GET %}
+    <div id="google_query">{% trans "Query results for: " %}<b>{{ request.GET.q }}</b></div>
```
Comment (mitar) [inline piplmesh/frontend/templates/search.html:9]: Indentation. Don't put end space into translatable string.

Comment (domenc) [inline piplmesh/frontend/templates/search.html:9]: Okey

**Comment by domenc** [comment]
Adding (my mistake - it was very late for me)

File: piplmesh/frontend/static/piplmesh/css/style.css (hunk 1 above)
Comment (domenc) [inline piplmesh/frontend/static/piplmesh/css/style.css:98]: Testing the right position on page..

**Comment by mitar** [comment]
Please create new pull request with commit with correct commit message.

**Pull request closed without merging**